services/web-app/components/markdown/image.js

```diff
@@ -24,8 +26,11 @@ const getImageProps = (props) => {
     props.layout = 'responsive'
   }
 
-  // do not optimize if is svg file
-  if (src.split('/').pop().includes('.svg')) {
+  // do not optimize if is svg file (absolute or relative)
+  if (
+    src.endsWith('.svg') ||
+    (isAbsoluteUrl(src) && new URL(src).pathname.split('/').pop().endsWith('.svg'))
+  ) {
```

Comment:
I know this is tempting for brevity, but we should not instantiate objects inside of a conditional expressions. The new url should be defined above instead.

Reply:
done